experimental/add_to_app/README.md

```diff
@@ -20,12 +24,19 @@ There are two Flutter modules included in the codebase:
 * `flutter_module` displays the dimensions of the screen, a button that
   increments a simple counter, and an optional exit button.
 * `flutter_module_using_plugin` does everything `flutter_module` does, and adds
-  another button that will save the counter's value to a file using the
-  [`path_provider`](https://pub.dev/packages/path_povider) Flutter plugin.
+  another button that will open the Flutter documentation in a browser using the
+  [`url_launcher`](https://pub.dev/packages/url_launcher) Flutter plugin.
 
-Before running any of the Android or iOS apps included in this sample project,
-you first need to resolve the Flutter modules' depencies. Do so by running this
-command from within the `flutter_module` and `flutter_module_using_plugin`
+Both of these modules feature the latest Flutter embedding API. As a result,
+prior to working with them, you need to configure your local installation of
+Flutter to use the new API as well. To do so, run this command:
+
+```
+flutter config --enable-android-embedding-v2
```

Comment:
@blasten I think in a week, we're going to turn this on right. Maybe we don't need to document this.

Reply:
+1

Reply:
Done.